packages/orchestrator-ui-components/src/components/WFOSubscription/styles.ts

```diff
@@ -0,0 +1,55 @@
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+
+import { css } from '@emotion/react';
+
+export const getStyles = (theme: EuiThemeComputed) => {
+    const contentCellStyle = {
+        padding: theme.base,
+        borderBottom: theme.border.thin,
+    };
+
+    const headerCellStyle = css({
+        ...contentCellStyle,
+        paddingLeft: 0,
+        width: '250px',
```

Comment:
I think it is better to use a multiple of base value, even when you cannot make it exactly 250. Base*16 would work too i assume?

Reply:
updated